src/module_wrap.h

```diff
@@ -59,8 +66,11 @@ class ModuleWrap : public BaseObject {
       v8::Local<v8::Context> context,
       v8::Local<v8::String> specifier,
       v8::Local<v8::Module> referrer);
-  static ModuleWrap* GetFromModule(node::Environment*, v8::Local<v8::Module>);
+  static ModuleWrap* GetFromModule(
+      v8::Local<v8::Context>,
+      v8::Local<v8::Module>);
 
+  static int Identity_;
```

Comment:
As I said before, we should try to reduce the number of thread-unsafe global state. This should go into `Environment`.